new_webapp/src/components/ExampleTable.js

```diff
@@ -86,8 +91,8 @@ const ExampleTableBase = ({
               <EditingState
                 editingRowId={editingRowId}
                 onEditingCellsChange={setEditingRowId}
-                addedRows={[]}
-                onAddedRowsChange={addEmptyRow}
+                addedRows={addedRows}
+				onAddedRowsChange={changeAddedRows}
```

Comment:
Styling, this one really bothers me, could you align `onAddedRowsChange` with the rest of the attributes?

Also, this makes `addEmptyRow` dead code, could you delete it please!

Reply:
https://www.youtube.com/watch?v=cowtgmZuai0 :P